packages/nextcloud/test/user_status_test.dart

```diff
@@ -11,17 +11,22 @@ void main() {
     (final preset) {
       late DockerContainer container;
       late NextcloudClient client;
-      setUp(() async {
+      setUpAll(() async {
         container = await DockerContainer.create(preset);
         client = await TestNextcloudClient.create(container);
       });
-      tearDown(() async {
+      tearDownAll(() async {
         if (Invoker.current!.liveTest.errors.isNotEmpty) {
           print(await container.allLogs());
         }
         container.destroy();
       });
 
+      Future<void> resetStatus() async {
+        await client.userStatus.userStatus.setStatus(statusType: 'online');
+        await client.userStatus.userStatus.clearMessage();
+      }
```

Comment:
i think in these cases the reset can be called in a `setup` hook so we don't need to explicitly add it everywhere

Reply:
Yes, but the problem here was that not all tests need it and I wanted to avoid unnecessary resets. Using a group would fix it, but then the grouping with the controllers/tags is not in place anymore. Not sure which option is better.

Reply:
In this case we can probably leave it. I prefer the grouping as is.